.github/workflows/ci-test-controllers.yml

```diff
@@ -80,7 +80,10 @@ jobs:
             kubectl apply -f -
           fi
           kubectl wait -n kubearmor --timeout=5m --for=jsonpath='{.status.phase}'=Running kubearmorconfigs/kubearmorconfig-test
-          kubectl wait --timeout=7m --for=condition=ready pod -l kubearmor-app,kubearmor-app!=kubearmor-snitch -n kubearmor
+          while true; do
+            kubectl get pod -n kubearmor -l kubearmor-app,kubearmor-app!=kubearmor-snitch | grep Running > /dev/null && break || echo "Waiting for kubearmor pods"
+            sleep 1
+          done
```

Comment:
Let's add a timeout on this step because it's an infinite loop

We should break early if something is in CrashLoopBackOff